<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapEventPage</class>
 <widget class="QWidget" name="MapEventPage">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,20">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="conditionsBox">
          <property name="title">
           <string>Conditions</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="verticalSpacing">
              <number>3</number>
             </property>
             <item row="1" column="0">
              <widget class="SelectorButton" name="switch1Button">
               <property name="text">
                <string notr="true">...</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="SelectorButton" name="switch2Button">
               <property name="text">
                <string notr="true">...</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="SelectorButton" name="itemButton">
               <property name="text">
                <string notr="true">...</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QCheckBox" name="switch1CheckBox">
               <property name="text">
                <string>Switch</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="switch2CheckBox">
               <property name="text">
                <string>Switch</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QCheckBox" name="variableCheckBox">
               <property name="text">
                <string>Variable</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QCheckBox" name="actorCheckBox">
               <property name="text">
                <string>Actor</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QCheckBox" name="itemCheckBox">
               <property name="text">
                <string>Item</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QPushButton" name="actorButton">
               <property name="text">
                <string notr="true">...</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QCheckBox" name="selfSwitchCheckBox">
                 <property name="text">
                  <string>Self Switch</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="selfSwitchComboBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>48</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <item>
                  <property name="text">
                   <string notr="true">A</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string notr="true">B</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string notr="true">C</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string notr="true">D</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item row="3" column="0" colspan="2">
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="SelectorButton" name="variableButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string notr="true">...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string notr="true"> &gt;= </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="variableSpinBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>0</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QGroupBox" name="imageBox">
            <property name="title">
             <string>Image</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="leftMargin">
              <number>4</number>
             </property>
             <property name="topMargin">
              <number>4</number>
             </property>
             <property name="rightMargin">
              <number>4</number>
             </property>
             <property name="bottomMargin">
              <number>4</number>
             </property>
             <item>
              <widget class="ClickableLabel" name="imageLabel">
               <property name="minimumSize">
                <size>
                 <width>64</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="movementBox">
            <property name="title">
             <string>Movement</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="spacing">
              <number>3</number>
             </property>
             <property name="leftMargin">
              <number>4</number>
             </property>
             <property name="topMargin">
              <number>4</number>
             </property>
             <property name="rightMargin">
              <number>4</number>
             </property>
             <property name="bottomMargin">
              <number>4</number>
             </property>
             <item>
              <layout class="QGridLayout" name="gridLayout_2">
               <property name="verticalSpacing">
                <number>3</number>
               </property>
               <item row="2" column="1">
                <widget class="QComboBox" name="movementSpeedComboBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <item>
                  <property name="text">
                   <string>x8 Slower</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>x4 Slower</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>x2 Slower</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Normal</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>x2 Faster</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>x4 Faster</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="movementRouteButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="movementTypeComboBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <item>
                  <property name="text">
                   <string>Fixed</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Random</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Approach</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Custom</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Type</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QComboBox" name="movementFreqComboBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <item>
                  <property name="text">
                   <string>Lowest</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Lower</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Normal</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Higher</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Highest</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Speed</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Freq</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QGroupBox" name="optionsBox">
            <property name="title">
             <string>Options</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>4</number>
             </property>
             <property name="topMargin">
              <number>4</number>
             </property>
             <property name="rightMargin">
              <number>4</number>
             </property>
             <property name="bottomMargin">
              <number>4</number>
             </property>
             <item>
              <widget class="QCheckBox" name="walkingCheckBox">
               <property name="text">
                <string>Walking</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="steppingCheckBox">
               <property name="text">
                <string>Stepping</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="directionFixCheckBox">
               <property name="text">
                <string>Direction Fix</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="throughCheckBox">
               <property name="text">
                <string>Through</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="optionsBox_2">
            <property name="title">
             <string>Other</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>4</number>
             </property>
             <property name="topMargin">
              <number>4</number>
             </property>
             <property name="rightMargin">
              <number>4</number>
             </property>
             <property name="bottomMargin">
              <number>4</number>
             </property>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Priority</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="priorityComboBox">
               <item>
                <property name="text">
                 <string>Below characters</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Same as characters</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Above characters</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Trigger</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="triggerComboBox">
               <item>
                <property name="text">
                 <string>Action button</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Player touch</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Event touch</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Autorun</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Parallel</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="contentBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Content</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="EventContentList" name="eventContentList">
        <property name="font">
         <font>
          <family>Ubuntu Mono</family>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>clickable_label.hpp</header>
  </customwidget>
  <customwidget>
   <class>SelectorButton</class>
   <extends>QPushButton</extends>
   <header>selector_button.hpp</header>
  </customwidget>
  <customwidget>
   <class>EventContentList</class>
   <extends>QListView</extends>
   <header>event_content_list.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
